domains/transfer-agreements/TransferAgreements.API/API/swagger.io.apibeskrivelse.txt

```diff
@@ -0,0 +1,184 @@
+openapi: 3.0.3
```

Comment:
I know Christoffer wants this file here, but I think it should not be here. Or be in a different PR where the entire API is actually specified. 
But it is up to you to do what you want. You are welcome to ignore my comment here and click "resolve conversation".